scoring/general-helpers.metta

```diff
@@ -0,0 +1,112 @@
+
+
+(= (addToSpace $row $space) (add-atom $space $row))
+
+(= (removeFromSpace $row $space) (remove-atom $space $row))
```

Comment:
Same comment here. I still think this can be done using only the `remove-atom` function. Don't you think?